examples/asr/speech_to_text_quant_infer.py

```diff
@@ -21,13 +21,21 @@
 from pprint import pprint
 
 import torch
-from pytorch_quantization import nn as quant_nn
 from pytorch_quantization import quant_modules
```

Comment:
There's an additional import here

Reply:
black moved things around, the check was on the first import. will move all the pytorch_quantization import under the try clause.